.env.example

```
@@ -0,0 +1,21 @@
+# Discord Bot
+DISCORD_TOKEN=your_bot_token_here
+CLIENT_ID=your_client_id_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLIENT_ID key should go before the DISCORD_TOKEN key

(UnorderedKey)